app/_how-tos/generate-certificate-with-sni.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Associate a Certificate with an SNI
+content_type: how_to
+related_resources:
+  - text: SNI entity
+    url: /gateway/entities/snis/
+  - text: Certificates
+    url: /gateway/entities/certificate/
+
+
+products:
+    - gateway
+tier: enterprise
+
+works_on:
+    - on-prem
+
+entities: 
+  - certificate
+  - sni
+  - route
+  - service
+
+tldr:
+    q: How do you validate that a Certificate is attached to an SNI and working?
+    a: After creating the Certificate and associating it to the desired SNI, then using `openssl` you can verify that {{site.base_gateway}} is returning the expected certificate for the SNI. 
+tools:
+    - deck
+
+prereqs:
+  inline:
+    - title: Certificate
+      include_content: prereqs/certificate
+      icon_url: /assets/icons/file.svg
+    - title: Configure environment variables
+      content: |
+        Set the following variables: 
+        * `CERT_ID`: The UUID set when you associated a Certificate with {{site.base_gateway}}.
+        * `SNI_NAME`: The name of the [SNI](/gateway/entities/sni/)
+        For example: 
+        ```sh
+        export CERT_ID=3f19b7d1-705f-422a-a116-7dc8282efe21
+        export SNI_NAME=my.sni.example.com
+        ```
+    
+cleanup:
+  inline:
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg
+    
+
+min_version:
+    gateway: '3.4'
+---
+
+
+## 1. Associate your certificate with an SNI
+
+Associate the Certificate created as a prerequisite with an SNI. If the SNI doesn't exist, this request will create a new one.
+
+{% capture request %}
+{% control_plane_request %}
```

Comment:
Why write this in curl? You can do this in decK as well. I would do it like we have it documented here: https://kongdeveloper.netlify.app/gateway/entities/sni/#set-up-an-sni